### DISCUSSION TITLE
Request to list subjects, biosamples and files under the Project details

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#272] [Feature Requests]
Dear CFDE-CC team:

I am from Metabolomics Workbench DCC. Earlier, when browsing the projects (at least for the MW DCC), in the details page I could see a list of subjects, biosamples and files associated with that project. Now, I don't see it. It would be great to have that feature back, if it doesn't require extensive effort. In fact, I used to use that feature to do some simple cross-checks during metadata upload, e.g., do the samples for that project on CFDE portal and the Metabolomics Workbench match?

Examples: https://app.nih-cfde.org/chaise/record/#1/CFDE:project/nid=77
or
https://app.nih-cfde.org/chaise/record/#1/CFDE:project/nid=98

Thank you very much.

Mano

**Comment by raynamharris** [comment]
I agree that this would be a very useful feature.
